File: Largest Prime Factor/largest_prime.c
#include <stdio.h>
#include <math.h>
int main() { 
long largest_prime=0;
int chekcer=0;
long number=floor(sqrt(600851475143)); //Marks Top Boundary 
for(long ii=13;ii<number;ii+=2){
    printf ("%ld\n",ii);
    if (600851475143%ii==0){        //Use Original number
        chekcer=1;
        for (long j=2;j<ii;j++){
            if (ii%j==0){
                chekcer=0;
                break;
            }
        }
        if (chekcer){
            largest_prime=ii;
        }
            
     }
        
}
printf ("The largest prime is %ld\n",largest_prime);       

}
            

File: Largest Prime Factor/largest_prime.py
import math
largest_prime=0
chekcer=False
for x in range (12,600851475143):
    if x%600851475143==0:
        chekcer=True
        for i in range (2,x):
            if i%x==0:
                chekcer=False
                break
        if chekcer:
            largest_prime=x

print (largest_prime)                
            